kube-derive/src/custom_resource.rs

```diff
@@ -238,7 +238,7 @@ impl CustomDerive for CustomResource {
         let has_status = status.is_some();
 
         let root_obj = quote! {
-            #[derive(Serialize, Deserialize, Clone, Debug)]
+            #[derive(kube::serde::Serialize, kube::serde::Deserialize, Clone, Debug)]
```

Comment:
Hm. This would make using `kube_derive` without `kube` impossible. I'd rather make `serde_json` an explicit dependency of `kube-derive` and re-export from there.